utils/skiptls.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import (
+	"crypto/tls"
+	"net/http"
+	"os"
+)
+
+var GOTIFY_SKIP_TLS = "GOTIFY_SKIP_TLS"
+
+func ShouldSkipTLSchecking() {
+	should_skip_tls := os.Getenv(GOTIFY_SKIP_TLS) == "True"
+
+	if should_skip_tls {
+		http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	}
+}
```

Comment:
Changing the default transport isn't that nice, as it may affects programs that use the cli as dependency.

Creating a http client with the insecure flag is a little cleaner. This client must be used every time gotify.NewClient is called.

Can we also rename the name of the env variable to `GOTIFY_SKIP_VERIFY_TLS`. Because it doesn't skip tls it only skips the verification of the certificate. Also, it would be cooler, if the value of the env var would be case insensitive.
```suggestion
func CreateHTTPClient() *http.Client {
	skipVerify := strings.ToLowerCase(os.Getenv("GOTIFY_SKIP_VERIFY_TLS")) == "true"

	customTransport := http.DefaultTransport.(*http.Transport).Clone()
	customTransport.TLSClientConfig = &tls.Config{InsecureSkipVerify: skipVerify}

	return &http.Client{Transport: customTransport}
}
```

Reply:
Thanks @jmattheis , makes total sense.
I am hoping not to take too much of your time here :( , have addressed this the way I hope you meant.

Reply:
Don't worry, reviews are a learning experience for both of us :D.